opal/core/search/queries.py

```diff
@@ -10,6 +10,7 @@
 from opal import models
 from opal.core import fields, subrecords
 from opal.utils import stringport
+from opal.core.search.search_query import SearchQuery
 
```

Comment:
`opal.core.search.queries` and `opal.core.search_query` ? 

I guarantee I won't remember the distinction in 6m :)